src/URIUtils.js

```diff
@@ -4,17 +4,3 @@
  * SPDX-License-Identifier: MIT
  */
```

Comment:
What's the reason for keeping the empty file?

Reply:
Oops, no!